alf/trainers/off_policy_trainer.py

```diff
@@ -124,21 +132,37 @@ def train_():
         env.reset()
         time_step = driver.get_initial_time_step()
         policy_state = driver.get_initial_state()
+
+        collect_steps = initial_collect_steps
+        if not collect_steps:
+            collect_steps = num_steps_per_iter
+
+        time_step, policy_state = driver.run(
+            max_num_steps=collect_steps,
+            time_step=time_step,
+            policy_state=policy_state)
+
         for iter in range(num_iterations):
             t0 = time.time()
-            time_step, policy_state = driver.run(
-                max_num_steps=num_steps_per_iter,
```

Comment:
max_num_steps=num_steps_per_iter if iter > 0 else collect_steps, so that we don't need two driver.run

Reply:
updated